**Wednesday 6/20**

**Boehm Talk**

* Pliertropy vs LD.  Interesting.  see hand written notes

**Thursday 6/21**

**Broman Talk**

* heterogeneous stock: one of 8 alleles
* worried about "bad" data
* QC process
  * Missing data (percent missing data on the mouse/sample level first)
      * more than 10% missing remove
  * Discoordinate Sex Prediction
    * average Y and average X intensity (from SNP arrays)
    * some indication of a X0 mouse (females with only 1 X chr), thinks they are fine samples
    * for SNPs, plots average X chr intensity by proportion of heterogygousity on X.  Males should have no heterpzygosity
  * Sample Duplicates
    * percent matching genotypes between pairs... histogram based on each pairwise comparison
    * look for samples at 100%
  * Sample mixups
    * look at genes with strong eQTLs, get a predicted genotype for each mouse/sample (and also predicted gene expression)
    * matrix of DNA sample by RNA sample (distance matrix)
    * should all be on diagonal.  Can't tell if RNA or DNA swapped.
  * Look into genotype errors based on genotypes from DNA seq and microbiome
  * Can construct genome where each genotype came from what parent by number of cross-overs
    * number of cross-overs increase in each generation
  * Look at quality of markers
    * can be wrong locations as well

**Perry Talk**

* how to identify candidate genes from QTL analyses
* integrate data via MGI (from JAX lab)
* zoned out a bit...

**Le Talk**

* Imputation for low coverage 
* HS population. 
* Have haplotype reference panel (HRP).  From the founder 8 strains.
* Get low-coverage DNA-seq to imput instead of HRP
* use the STITCH algorithm
* Able to detect stronger QTLs using the imputted data 

**Prins Talk**






    
